cfgov/jinja2/v1/events/_macros.html

```diff
@@ -344,42 +231,37 @@ <h2 class="event-venue_heading">{{ city }}</h2>
             </tr>
         </thead>
         <tbody>
-        {% for block in post.agenda %}
+        {% for block in event.agenda_items %}
+          {% set bound = block.value.bound_blocks %}
             <tr>
                 <td class="{{ options.time_col_classes }}">
                     {% import 'macros/time.html' as time %}
-                    {{ time.render(block.beginning_time.date,
-                                   {'time':true, 'timezone':true},
-                                   block.beginning_time.timezone) }}
+                    {{ time.render(bound.start_dt.render(), {'time':true, 'timezone':true}) }}
                     &ndash;
-                    {{ time.render(block.ending_time.date,
-                                   {'time':true, 'timezone':true},
-                                   block.ending_time.timezone) }}
+                    {{ time.render(bound.end_dt.render(), {'time':true, 'timezone':true}) }}
                 </td>
                 <td class="{{ options.agenda_col_classes }} simple-table_row-heading">
-                    {{ block.desc }}
+                    {{ bound.description.render() }}
                 </td>
                 <td class="{{ options.location_col_classes }}" data-label="Location">
-                  {% if block.location %}
-                    {{ block.location}}
+                  {% if bound.location.render() %}
+                    {{ bound.location.render()}}
                   {% endif %}
                 </td>
                 <td class='{{ options.speaker_col_classes }}' data-label="Speakers">
-                {% for speaker in block.speakers %}
-                    {%- if speaker.url -%}
-                    <a href="{{ speaker.url }}">
-                    {{ speaker.name
-                       ~ ( ', ' if speaker.name.find(',') == -1 )
-                       ~ speaker.position
+                {% for speaker in bound.speakers.value %}
+                    {%- if speaker.bound_blocks.url -%}
+                    <a href="{{ speaker.bound_blocks.url.render() }}">
+                    {{ speaker.bound_blocks.name.render()
+                       ~ ( ', ' if speaker.bound_blocks.name.value.find(',') == -1 else '' )
                     }}
                     </a>
                     {%- else -%}
-                    {{ speaker.name
-                       ~ ( ', ' if speaker.name.find(',') == -1 )
-                       ~ speaker.position
+                    {{ speaker.bound_blocks.name.render()
+                       ~ ( ', ' if speaker.bound_blocks.name.value.find(',') == -1 else '' )
```

Comment:
:+1: Cool yeah, it was just that this whole line was repeated a few lines earlier and it looked kinda complex so could maybe be moved to a helper to make it clearer what was happening. But replace works.